actions/ajax/deadline_extension/new.php

```diff
@@ -28,24 +28,24 @@
 $allocatabletype = optional_param('allocatabletype', $USER->id, PARAM_ALPHANUMEXT);
 $requesttype = optional_param('requesttype', 'new', PARAM_ALPHANUMEXT);
 $id = optional_param('id', 0, PARAM_INT);
-$extra_information_text = optional_param('text', '', PARAM_RAW);
-$extra_information_format = optional_param('format', 1, PARAM_ALPHANUMEXT);
-$extended_deadline = optional_param('extended_deadline', 0, PARAM_ALPHANUMEXT);
-$pre_defined_reason = optional_param('pre_defined_reason', null, PARAM_ALPHANUMEXT);
+$extrainformationtext = optional_param('text', '', PARAM_RAW);
```

Comment:
but I assume this param is used on line 45 without the the name of the param being consistent?

Reply:
I think it's ok that the var names like $extrainformationtext are changed while the corresponding keys within the params array like $params['extra_information_text'] are kept with underscores.  Those keys could end up being used elsewhere e.g. in this case it looks like the code is putting together a response to an AJAX request and the JS at the other end is likely expecting underscores in the response.  It looks like phpcbf also ignores underscores in object keys like $a->test_thing in the same way which also seems correct to me